extensions/pkg/controller/worker/genericactuator/actuator_reconcile.go

```diff
@@ -290,7 +312,8 @@ func (a *genericActuator) waitUntilWantedMachineDeploymentsAvailable(ctx context
 	log.Info("Waiting until wanted machine deployments are available")
 
 	return retryutils.UntilTimeout(ctx, 5*time.Second, 5*time.Minute, func(ctx context.Context) (bool, error) {
-		var numHealthyDeployments, numUpdated, numAvailable, numUnavailable, numDesired, numberOfAwakeMachines int32
+		var numHealthyDeployments, numUpdated, numAvailable, numUnavailable, numDesired, numberOfAwakeMachines,
+			numDesiredManualInPlace, numUpdatedManualInPlace, numNeedUpdatedManualInPlace, numOldMachinesNotUpdateCandidateManualInPlace int32
```

Comment:
```suggestion
			numNeedUpdatedManualInPlace, numOldMachinesNotUpdateCandidateManualInPlace int32
```

I don't see the need for these two variables.